chef_master/source/config_yml_kitchen.rst

```diff
@@ -133,7 +133,7 @@ Kitchen can configure the chef-zero provisioner with the following Chef-specific
    * - ``chef_metadata_url``
      - **This will be deprecated in a future version.**
    * - ``chef_omnibus_install_options``
-     - Use to specify the package to be installed. Possible values: ``-P chef`` (for Chef Infra Client) and ``-P chefdk`` (for the Chef Infra Client that is packaged as part of ChefDK). Use ``-n`` to specify the nightly build. For example: ``-P chefdk`` or ``-n -P chefdk``. **This will be deprecated in a future version.** See the ``product_name``, ``product_version``, and ``channel`` settings instead.
+     - Use to specify the package to be installed. Possible values: ``-P chef`` (for Chef Infra Client) and ``-P chef-workstation`` (for the Chef Infra Client that is packaged as part of Chef Workstation). Use ``-n`` to specify the nightly build. For example: ``-P chef-workstation`` or ``-n -P chef-workstation``. **This will be deprecated in a future version.** See the ``product_name``, ``product_version``, and ``channel`` settings instead.
```

Comment:
Is the bolded deprecation note still applicable with this change from DK -> workstation?

Reply:
Should still be valid